docs/AppService/PublishAppService/assets/PublishAppService.puml

```diff
@@ -1,10 +1,9 @@
 @startuml PublishAppService
 participant HMIProvider
 participant Core
-participant AppServiceManager
 
-HMIProvider -> Core: AppService.PublishAppService
-Core -> AppServiceManager: PublishAppService
-Core --> HMIProvider: AppService.PublishAppService Response
+HMIProvider -> Core: AppService.PublishAppService\n(MEDIA, "HMI Media Service")
+Core --> HMIProvider: AppService.PublishAppService Response\n(serviceId: "service1")
+Core ->> HMIProvider : BasicCommunication.OnSystemCapabilityUpdated\n(APP_SERVICES, PUBLISHED)
```

Comment:
I'll note that `OnSystemCapabilityUpdated` is sent by Core before it sends a `PublishAppService` response